src/services/mentors.js

```diff
@@ -577,13 +578,35 @@ module.exports = class MentorsHelper {
 
 			const totalSession = await sessionAttendeesQueries.countEnrolledSessions(id)
 
+			const userRoleIds = roles.map((role) => role.id)
+			const filter = { role_id: userRoleIds }
+			const permissionAndModules = await rolePermissionMappingQueries.find(filter)
+			const permissionsByModule = {}
+
+			permissionAndModules.forEach((rolePermission) => {
+				const module = rolePermission.dataValues.module
+				const actions = rolePermission.dataValues.actions
```

Comment:
@sumanvpacewisdom please do the request_type change, where ever you are referring to the actions